package jedi.projeto.primeApp.exception;

public class RecursoNaoEncontradoException extends RuntimeException {

    public RecursoNaoEncontradoException() {
        super("Recurso n√£o encontrado");
    }
    public RecursoNaoEncontradoException(String mensagem){
    super(mensagem);

    }

}
